systemtest/src/test/java/io/strimzi/systemtest/SecurityST.java

```diff
@@ -424,7 +422,7 @@ void autoReplaceSomeKeysTriggeredByAnno(final List<String> secrets,
             internalKafkaClient.receiveMessagesPlain()
         );
 
-        SecretUtils.waitForSecretReady(bobUserName);
+        SecretUtils.waitForSecretReady(user.getMetadata().getName());
```

Comment:
I don't think that's really needed to wait for secret is ready here - also it's not needed after user creation, because we check this when we are creating the `KafkaUser`